### DISCUSSION TITLE
Adjusting PRB based on sst

### DISCUSSION THREAD

**Discussion by hatim4** [srsran/oran-sc-ric#32] [Q&A]
Hi all,

I tried the RC xApp example and it is working with no issue (Open5GS + srsRAN Project + oran-sc-ric). It must be given a UE ID (required by the RC ControlHeader). Is it possible to adjust the PRB based on pre-defined slices (sst number) rather than per UE ID? Because as far as I know we cannot get the UE IMSI from gNB to control the PRB of a specific UE through the RC xApp.

Thanks.

**Comment by pgawlowicz** [comment]
currently, it is not supported.
